src/sagas/login.js

```diff
@@ -173,106 +138,83 @@ function* checkOvirtApiVersion (oVirtMeta) {
   return passed
 }
 
-export function* logout () {
-  if (AppConfiguration.applicationLogoutURL && AppConfiguration.applicationLogoutURL.length > 0) {
-    window.location.href = AppConfiguration.applicationLogoutURL
+function composeIncompatibleOVirtApiVersionMessage (oVirtMeta) {
```

Comment:
I think this function must be in `utils` too

Reply:
Perhaps, but it is only ever used by the `login()`, and only if the version check fails.  I don't see the function being used anywhere else, so I went with keeping it here.

Reply:
As well as `doCheckTokenExpired`, that used only in `index.js`. So I still think that better move it to `utils`.

Reply:
Any `doCheckTokenExpired` and `callExternalAction` work should probably be done under the context of #991.  I'll leave tweaking these functions for that PR.